### PR TITLE
Add Portuguese translations

### PR DESCRIPTION
**Pull request by abraaolevi**
Adding translations in Portuguese for the theme.

### CODE REVIEW

File: i18n/pt.yaml [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+- id: noposts_warning_description
+  translation:
+    "<p>Quando você criar uma postagem em qualquer pasta (seção) dentro do diretório content, ele vai aparecer aqui.</p> \
+    <p>Apenas uma seção (a que conter mais postagens) será exibida na página principal por padrão.</p>"
+
+- id: noposts_warning_tip
+  translation:
+    "<b>Dica:</b> Você pode mostrar quantas seções você quiser com o parâmetro de configuração \
+    <b><a href=\"https://gohugo.io/functions/where/#mainsections\" rel=\"nofollow noopener\" target=\"_blank\">mainSections</a></b>."
```
Comment (Vimux) [inline i18n/pt.yaml]:
```suggestion
    <b><a href=\"https://gohugo.io/functions/where/#mainsections\" rel=\"nofollow noopener\" target=\"_blank\">mainSections</a></b>."

```

Final newline

File: i18n/pt.yaml [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+  translation: "página principal"
+
+# No posts empty state
+- id: noposts_warning_title
+  translation: "Você ainda não tem nenhuma postagem!"
+
+- id: noposts_warning_description
+  translation:
+    "<p>Quando você criar uma postagem em qualquer pasta (seção) dentro do diretório content, ele vai aparecer aqui.</p> \
+    <p>Apenas uma seção (a que conter mais postagens) será exibida na página principal por padrão.</p>"
```
Comment (Vimux) [inline i18n/pt.yaml]: There is no need to use `p` tags here.

File: i18n/pt-br.yaml [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+  translation: "página principal"
+
+# No posts empty state
+- id: noposts_warning_title
+  translation: "Você ainda não tem nenhuma postagem!"
+
+- id: noposts_warning_description
+  translation:
+    "<p>Quando você criar uma postagem em qualquer pasta (seção) dentro do diretório content, ele vai aparecer aqui.</p> \
+    <p>Apenas uma seção (a que conter mais postagens) será exibida na página principal por padrão.</p>"
```
Comment (Vimux) [inline i18n/pt-br.yaml]: There is no need to use `p` tags here.

Comment (abraaolevi) [reply i18n/pt-br.yaml]: Thank you for review. I made changes requested :)

**Pull request merged**